### PR TITLE
Create FUNDING.json

### PR DESCRIPTION
**Pull request by xdaluca**
FUNDING.json needed to claim funds on drips.network. [Click here](https://docs.drips.network/claim-your-repository#how-drips-manages-project-access) for more details.

This should stay in the root of the repo, e.g. [ethers.js](https://github.com/ethers-io/ethers.js/blob/main/FUNDING.json)

### CODE REVIEW

**Comment by Amxx** [comment]
Do you know if drips support ens addresses ? If so, I think we should setup an ENS address for that. 
I'm honestly not confortable having an ethereum address I cannot verify here.

**Comment by ernestognw** [comment]
Big ++ to @Amxx

**Comment by xdaluca** [comment]
I agree that ENS would be better, I asked the drips team, and unfortunately it does not support ENS for now. However, the current wallet is a SAFE so if there was switching around of wallets it would already be easier.

We could link an ENS subdomain to this SAFE however.

**Review by ernestognw — APPROVED** [review]
Ok, LGTM! Sorry for taking some time, I went through Drips documentation as well

**Comment by xdaluca** [comment]
@ernestognw thanks! do PRs get merged on schedule?

**Comment by ernestognw** [comment]
> @ernestognw thanks! do PRs get merged on schedule?

I was waiting on @Amxx approval. I think we can merge because it's not critical. However, can we have the Safe owners set up before merging?

**Comment by xdaluca** [comment]
@ernestognw my bad I forgot. Safe owners are set.

**Comment by ernestognw** [comment]
Cool, I think the setup is good for now to merge.

**Pull request merged**
